### DISCUSSION TITLE
making the tracker faster

### DISCUSSION THREAD

**Discussion by Elad-Shmueli** [STVIR/pysot#557] [Code]
i was hoping someone could help me push faster processing from the trackers with the cost of lower accuracy.
i am not sure about the difference in models and configs and backbones but alex model runs fastest on my machine(jetson xavier NX),
it runs on about 30~fps, but i want more, and i dont mind losing performence.
can anyone please tell me what i can do to make it run faster, do i need to train the tracker on a small data set? or make the program use more GPU power?
any insight would be highly appreciated

**Comment by maidinaabuduxiku** [comment]
> i was hoping someone could help me push faster processing from the trackers with the cost of lower accuracy. i am not sure about the difference in models and configs and backbones but alex model runs fastest on my machine(jetson xavier NX), it runs on about 30~fps, but i want more, and i dont mind losing performence. can anyone please tell me what i can do to make it run faster, do i need to train the tracker on a small data set? or make the program use more GPU power? any insight would be highly appreciated

Hi，i have the same problem, have you improved your tracking speed? could your share your solutions please.
thank you a lot

**Reply by ZhiyuanChen** [reply]
Have you checked the opencv library? They do have SiamesRPN builtin
